generators/app/templates/resource/tasks/config/webpack.config.js

```diff
@@ -0,0 +1,142 @@
+const webpack = require('webpack'),
+  path = require('path'),
+  ExtractTextPlugin = require('extract-text-webpack-plugin');
+
+module.exports = {
+  // Where to start
+  entry: {
+    TagInput: path.resolve(__dirname, '../../src/index.js')
+  },
+
+  // Where to output
+  output: {
+    path: path.resolve(__dirname, '../../dist'),
+    filename: '[name].js',
+    libraryTarget: 'commonjs2'
+  },
+
+  externals: {
+    classnames: true,
+    react: 'React',
+    'react-dom': true,
+    'ship-components-outsideclick': true,
+    'ship-components-icon': true,
+    'react-addons-css-transition-group': true
+  },
+
+  module: {
+    rules: [
+      {
+        test: /\.(jsx?|es6)$/,
+        enforce: 'pre',
+        exclude: /(node_modules|dist)/,
+        include: /src\/.*/,
```

Comment:
I don't think we need to include and exclude do we? Can't we just exclude?

Reply:
That's what I thought, but last time I tried it was passing using both

Reply:
@ThinkTankShark It will pass, but it shouldn't be necessary. I believe recommendation is to use `include` which implicitly says "exclude everything else."

Reply:
Removes the exclude option